### PR TITLE
RWA-3130: Adding app-insight alert messages

### PR DESCRIPTION
**Pull request by kvvasudev**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RWA-3130

### Change description ###

Adding logs for app-insight alert messages

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x ] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/wacaseeventhandler/controllers/CaseEventHandlerReadinessHealthController.java [hunk 1]
```diff
@@ -32,6 +32,7 @@ protected AvailabilityState getState(ApplicationAvailability applicationAvailabi
         try {
             caseEventMessageCacheService.getAllMessagesInNewState(environment);
         } catch (Exception e) {
+            log.info("Readiness check failed with exception {} ", e.getMessage());
```
Comment (raghera) [inline src/main/java/uk/gov/hmcts/reform/wacaseeventhandler/controllers/CaseEventHandlerReadinessHealthController.java:35]: maybe at error or warn level would be better?

File: src/main/java/uk/gov/hmcts/reform/wacaseeventhandler/controllers/CaseEventHandlerLivenessHealthController.java [hunk 2]
```diff
@@ -62,6 +62,7 @@ protected AvailabilityState getState(ApplicationAvailability applicationAvailabi
                 && noOfNewMessages > newMessageStateThresholdForLivenessCheck
                 && allMessageInNewState.get(noOfNewMessages - 1).getReceived()
                     .isBefore(LocalDateTime.now().minusHours(numberOfHours))) {
+                log.info("Liveness check failed due to message readiness check failures");
```
Comment (raghera) [inline src/main/java/uk/gov/hmcts/reform/wacaseeventhandler/controllers/CaseEventHandlerLivenessHealthController.java:65]: error/warn level?

Comment (kvvasudev) [reply src/main/java/uk/gov/hmcts/reform/wacaseeventhandler/controllers/CaseEventHandlerLivenessHealthController.java]: Fixed

**Review by raghera — APPROVED** [review]
Looks good.

**Pull request merged**